.travis.yml

```diff
@@ -20,24 +20,29 @@ addons:
       - libatlas-base-dev
       - lcov
       - swig3.0
+      - python3-nbconvert # for jupyter notebooks
+      - jupyter-notebook  # for jupyter notebooks
+
 install:
   - pyenv shell 2.7 3.6
   - mkdir -p ~/bin/ && ln -s /usr/bin/swig3.0 ~/bin/swig && export PATH=~/bin/:$PATH
   - pip3 install --upgrade setuptools wheel pkgconfig
   - pip3 install --upgrade -r ./.travis_pip_reqs.txt
   - pip3 install .
+
 before_script:
   # use a no-screen x-server to test plotting
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3 # give xvfb some time to start
 
 # run tests
-
 script:  
   - python3 -m flake8 --ignore=F401 --exclude=build,doc,example
   - python3 -m pytest --cov=pypesto ./test
   - coverage xml
+  - test/runNotebook.sh doc/example/rosenbrock.ipynb
```

Comment:
would it be possible to change the script to only pass the directory, and then run all notebooks (*.ipynb) therein? would make it easier to catch up when notebooks are added.

Reply:
Can do, if you prefer. But I won't add an exclusion filter, in case you decide to not run all... ;)

Reply:
think will be better to test all. on that occasion, in conversion_reaciont.ipynb the profiles and sampling things need to be commented out, otherwise they won't pass.